shared/src/commonMain/kotlin/io.projectnewm.shared/login/LoginRepository.kt

```diff
@@ -0,0 +1,40 @@
+package io.projectnewm.shared.login
+
+//TODO: Handle Error Cases
+class LoginRepository(
+    private val service: LoginService
+) {
+    suspend fun registerUser(email: String): String {
+        return service.requestSignUpAccess(email)
```

Comment:
What is the string being returned here?  Also would have expected this function to also be called `requestEmailConfirmationCode` or something.

Reply:
The string being returned here would be a token, will update it once we have a usecase to test out the full integration with the UI